backend/api/Services/InspectionAreaService.cs

```diff
@@ -261,4 +371,25 @@ public void DetachTracking(FlotillaDbContext context, InspectionArea inspectionA
             context.Entry(inspectionArea).State = EntityState.Detached;
         }
     }
+
+    public class InspectionAreaPolygon
+    {
+        [JsonPropertyName("zmin")]
+        public double ZMin { get; set; }
+
+        [JsonPropertyName("zmax")]
+        public double ZMax { get; set; }
+
+        [JsonPropertyName("positions")]
+        public List<XYPosition> Positions { get; set; } = [];
+    }
```

Comment:
This could be moved to the database models folder, and made [Owned]. That way inspectionAreas could include it, but it would not make a new table.

Reply:
Actually, keeping it as a JSON could also be useful for simplifying adding new polygons. So I am OK with either approach.